manual/src/ornate/changelog.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Version 0.13.0
+
+> 2018 Jun 5
+
+The following PRs are included in this release
+
+- [#247](https://github.com/scalacenter/scalajs-bundler/pull/247): Assets to sbt
+
```

Comment:
What do you think about replacing the 0.13.0-RC1 section instead of creating a new 0.13.0 section?

Reply:
That's fine. I thought about doing that instead. I'll change it

Reply:
Updated